plugins/modules/user.py

```diff
@@ -368,13 +369,18 @@ def remove_user(module):
         Message for successfull command
     '''
     name = module.params['name']
-    cmd = ['userdel']
-
-    if module.params['remove_homedir']:
-        cmd.append('-r')
-
+    if module.params['load_module']:
```

Comment:
This will always be true, because the default value is "files". The else part will never run...